.zenodo.json

```diff
@@ -151,6 +151,9 @@
       "name": "Visconti di Oleggio Castello, Matteo",
       "orcid": "0000-0001-7931-5272"
     },
+    {
+      "name": "Michael Notter"
```

Comment:
@miykael Do you want to be on the Zenodo release? If so, should I use your [info from nipype](https://github.com/nipy/nipype/blob/master/.zenodo.json#L33-L37)?

Reply:
Hi @effigies. Yes, would be happy to be on the zenodo release, thank you! And yes, the info from nipype are still up to date.